tcs-persistence/src/main/resources/queries/traineeInfoForConnection.sql

```diff
@@ -18,6 +18,8 @@ from (
     -- note: null values are filtered out by the condition below
     and lower(gmc.gmcNumber) <> 'unknown'
     and gmc.gmcNumber not like CONCAT('%', UNHEX('c2a0'), '%') -- filter out all gmc number with non-breaking space
+  inner join Person p on (p.id = cd.id)
+    and LOWER(p.role) LIKE '%dummy%' OR LOWER(p.role) LIKE '%placeholder%'
```

Comment:
I think these can be removed.

Reply:
Oh i see my bad. Sorry I left it unintentionally

Reply:
Removed just now